src/bundle/Resources/config/views.yml

```diff
@@ -10,3 +10,8 @@ system:
                     template: "EzSystemsHybridPlatformUiBundle:content/tabs:details.html.twig"
                     controller: "EzSystemsHybridPlatformUiBundle:ContentView:detailsTab"
                     match: true
+            versions_tab:
+                default:
+                    template: "EzSystemsHybridPlatformUiBundle:content/tabs:versions.html.twig"
+                    controller: "EzSystemsHybridPlatformUiBundle:ContentView:versionsTab"
```

Comment:
On my instance, this change fixes the autowiring of the controller action arguments:

```
diff --git a/src/bundle/Resources/config/views.yml b/src/bundle/Resources/config/views.yml
index 5647921..12a09c5 100644
--- a/src/bundle/Resources/config/views.yml
+++ b/src/bundle/Resources/config/views.yml
@@ -13,5 +13,5 @@ system:
             versions_tab:
                 default:
                     template: "EzSystemsHybridPlatformUiBundle:content/tabs:versions.html.twig"
-                    controller: "EzSystemsHybridPlatformUiBundle:ContentView:versionsTab"
+                    controller: 'EzSystems\HybridPlatformUiBundle\Controller\ContentViewController:versionsTabAction'
                     match: true
```

According to the Sf doc, if the controller uses its class as its ID, the `Bundle:Controller:Action` syntax can be used. But it doesn't seem to behave as expected. The argument isn't filled in by the ArgumentResolver.